src/components/HostsTable.tsx

```diff
@@ -2,6 +2,7 @@ import React, { FC, Fragment } from 'react';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
 import { Bullseye } from '@patternfly/react-core';
 import { HostTableRows } from '../models/hosts';
+import { HostsEmptyState } from './HostsEmptyState';
```

Comment:
HostsEmptyState is default export, it needs to be imported as:
`import HostsEmptyState from './HostsEmptyState';`